clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -2064,6 +2064,9 @@ llvm::Value *CodeGenFunction::EmitToMemory(llvm::Value *Value, QualType Ty) {
 
   if (Ty->isExtVectorBoolType()) {
```

Comment:
you replaced a bunch of `isExtVectorBoolType()` with `isPackedVectorBoolType`. Why is this case different? Whats the difference between the two?